classes/mender-image.bbclass

```diff
@@ -9,4 +9,4 @@ inherit mender-image-buildinfo
 
 #Make sure we are creating sdimg with all needed partitioning.
 IMAGE_CLASSES += "mender-sdimg"
-IMAGE_FSTYPES ?= "tar.gz ext3 sdimg"
+IMAGE_FSTYPES_append = " sdimg"
```

Comment:
IMO this shouldn't be a large problem. The user can always not inherit mender-image, since the class is only useful when building the base/initial image.